training/src/main/java/wolox/training/exceptions/BookIdMismatchException.java

```diff
@@ -0,0 +1,25 @@
+package wolox.training.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.BAD_REQUEST, reason = "Actor Not Found")
```

Comment:
This reason does not correspond
```suggestion
@ResponseStatus(code = HttpStatus.BAD_REQUEST, reason = "Id mismatch")
```